tests/julia/julia_version-1.1/REQUIRE

```diff
@@ -0,0 +1,4 @@
+julia 1.1
+
+# Julia packages:
+Compat
```

Comment:
For my education: what is `Compat`? A while back we had a round of PRs to try and decrease the time it takes to run the whole test suite and I remember we were trying to find a Julia package to install that installs as quickly as possible and isn't totally unrealistic/fake/unmaintained. If you have any thoughts on that we could make a new issue/PR to change things. Keeping the tests fast is a constant struggle.

Reply:
This is a package that tries to backport changes in newer versions of julia to older versions of julia. It is well maintained, as far as I can tell, it doesn't have any dependencies, but it might be quite large.

I maintain https://github.com/queryverse/IteratorInterfaceExtensions.jl, and that must be a contender for the smallest package that is maintained. I don't know how much load time it would save, but probably a little bit...